src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -5011,4 +5008,19 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_TypeForwardedToMultipleAssemblies" xml:space="preserve">
     <value>Module '{0}' in assembly '{1}' is forwarding the type '{2}' to multiple assemblies: '{3}' and '{4}'.</value>
   </data>
+  <data name="ERR_BadDocumentationMode" xml:space="preserve">
+    <value>Provided Documentation Mode is unsupported or invalid: '{0}'.</value>
```

Comment:
Same.

---
In reply to: [101869014](https://github.com/dotnet/roslyn/pull/17113#discussion_r101869014) [](ancestors = 101869014)